base/uk.ac.stfc.isis.ibex.opis/resources/flipprps.opi

```diff
@@ -0,0 +1,276 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The OPI currently doesn't conform to our usual standards:
- Device name as OPI title
- `$(NAME)` as subtitle
- Items in grouping containers
- Fonts for labels etc

The easiest thing to do is usually to copy and change elements from other opis. There is a `template.opi` which describes most of our style guidelines as well as a wiki page